UserCard.jsx

```diff
@@ -0,0 +1,39 @@
+function UserCard({ userId }){
+  const [user, setUsers] = React.useState([]);
+  const [isLoading, setIsLoading] = React.useState(true)
+  
+  React.useEffect(()=>{
+    getUser(userId)  
+      .then(user =>{
+        setUsers(user);
+        setIsLoading(false);
+      });
+  },[]);
+  
+  
+  return (
+    <div>
+      <ul>
+      {isLoading ? ("...Loading"): 
```

Comment:
One other small comment - this works well and is very readable to have the `isLoading` state. Since you're really only waiting on the `user`, if you set the initial value of the user to `null`, you could also use that to determine if you should be "Loading" or not. Something like

```jsx
const [user, setUser] = useState(null);

// in the render

{ !user ? ("...Loading") : // render the rest of the component }
```

There's no right or wrong way to do it, just another idea that you can use. Feel free to choose whatever is more readable to you for your own projects!

Reply:
Yea I think this also reads more because we are waiting for the user to come back from the API and also would save some lines.